app/src/src/utils/endpoints.js

```diff
@@ -434,6 +434,26 @@ export class AuthEndpoint {
       }
     );
   }
+
+  /**
+   * Emails user password reset link.
+   * @param {string} email
+   * @return {Promise<AxiosResponse>}
+   * */
+  static async passwordReset(email) {
+    await removeAuthToken();
+    await removePushToken();
+    await removeUserData();
```

Comment:
im not sure if these are needed because it's  already handled on login and logout, and the password reset is only accessible before login